{
  "name": "rolly-retry",
  "version": "1.0.0",
  "description": "Retry function calls if they fail",
  "main": "dist/index.js",
  "module": "dist/index.d.js",
  "files": ["dist/**/*"],
  "scripts": {
    "test": "jest",
    "build": "tsc",
    "lint": "eslint"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/zzacal/rolly-retry.git"
  },
  "keywords": [
    "retry",
    "failover",
    "backoff",
    "error",
    "handling",
    "graceful"
  ],
  "author": "Zac Zacal <jizacal@gmail.com> (https://github.com/zzacal)",
  "license": "SEE LICENSE IN LICENSE",
  "bugs": {
    "url": "https://github.com/zzacal/rolly-retry/issues"
  },
  "homepage": "https://github.com/zzacal/rolly-retry#readme",
  "devDependencies": {
    "@types/jest": "^27.4.1",
    "@typescript-eslint/eslint-plugin": "^5.19.0",
    "@typescript-eslint/parser": "^5.19.0",
    "eslint": "^8.13.0",
    "ts-jest": "^27.1.4"
  }
}
